verif/regress/smoke-tests.sh

```diff
@@ -45,6 +45,8 @@ CC_OPTS="-static -mcmodel=medany -fvisibility=hidden -nostdlib -nostartfiles -g
 cd verif/sim/
 make -C ../.. clean
 make clean_all
+python3 cva6.py --testlist=../tests/testlist_riscv-compliance-cv32a60x.yaml --test rv32i-I-ADD-01 --iss_yaml cva6.yaml --target cv32a6_imac_sv32 --iss=$DV_SIMULATORS $DV_OPTS
+exit
```

Comment:
Foe master the test should not shortened like this.

Reply:
Yes, yes, this was intended for testing purposes only on CI as I did not have the error in local